core-bundle/config/services.yaml

```diff
@@ -744,6 +744,12 @@ services:
         tags:
             - { name: kernel.reset, method: reset }
 
+    contao.security.backend_favorites_voter:
+        class: Contao\CoreBundle\Security\Voter\DataContainer\BackendFavoritesVoter
```

Comment:
@Toflar Not sure if `Voter\DataContainer` is correct? It could also be just `Voter`.